R/build_GRAN_all.R

```diff
@@ -15,9 +15,16 @@ build_GRAN_all <- function(versions){
 	os = Sys.info()['sysname']
 	if (os == 'Darwin'){
 	  orig.version = dir_version()
+	  if(length(versions) > 1 && compareVersion(versions[1], versions[2]) == -1){
+	    stop("Currently only building source for 3.3, so 3.3 needs to build first")
+	  } 
 	  for (version in versions){
 		  set_version(version)
-			system("Rscript -e 'library(granbuild); dl_build_src(); build_bin()'")
+	    if(version == "3.2") {
```

Comment:
Well hopefully 3.3 won't start crashing the source build, but yeah it could be more robust.  I was trying to avoid having to write a function that sorts character version numbers, and `compareVersions` was the best I could find.

Although now that I think about it, since we aren't dealing with the patch numbers, we just enter the version numbers as numerics, sort them, and then convert to character